src/content/guides/getting-started.md

```diff
@@ -272,7 +272,7 @@ __package.json__
     "scripts": {
 -      "test": "echo \"Error: no test specified\" && exit 1" 
 +      "test": "echo \"Error: no test specified\" && exit 1",
-+      "build": "webpack"
++      "build": "webpack --mode=production"
```

Comment:
Having a warning is never a good thing, otherwise it's a `info` message.
More, people who discover webpack need to see what they must write to remove warning, why a build is for production whereas `"dev": "webpack --mode=development --watch"` with `--watch` is for development.
![image](https://user-images.githubusercontent.com/1462701/53793870-01ef8d00-3f2f-11e9-8eaf-1bf961542fb2.png)

Reply:
Hmm, maybe it is worth looking at changing its level `INFO` ?

Reply:
They maybe set it as warning for a reason.

Reply:
Looking at the guide, another thought, maybe add an info block e.g.
```
T> Ignore the [mode](/configuration/mode/) warning for now, we will get to it in the following guides.
```